Mathlib/CategoryTheory/Functor/Bifunctor.lean

```diff
@@ -37,12 +36,33 @@ abbrev Functor.mapₗ (F : C ⥤ D ⥤ E) {A B : C} (f : A ⟶ B) (X : D) :
     F.obj₂ A X ⟶ F.obj₂ B X :=
   (F.map f).app X
 
+@[simp, reassoc]
+lemma Functor.mapₗ_comp (F : C ⥤ D ⥤ E) {A B T : C} (f : A ⟶ B) (g : B ⟶ T) (X : D) :
+    F.mapₗ (f ≫ g) X = F.mapₗ f X ≫ F.mapₗ g X := by
+  simp [Functor.mapₗ]
+
+@[simp, reassoc]
+lemma Functor.mapₗ_id (F : C ⥤ D ⥤ E) {A : C} (X : D) :
+    F.mapₗ (𝟙 A) X = 𝟙 (F.obj₂ A X) := by
+  simp [Functor.mapₗ]
+
 /-- Action of two-variable functors on a morphism in the right argument. -/
 abbrev Functor.mapᵣ (F : C ⥤ D ⥤ E) (A : C) {X Y : D} (g : X ⟶ Y) :
     F.obj₂ A X ⟶ F.obj₂ A Y :=
   (F.obj A).map g
 
+@[simp, reassoc]
+lemma Functor.mapᵣ_comp (F : C ⥤ D ⥤ E) (A : C) {X Y Z : D} (f : X ⟶ Y) (g : Y ⟶ Z) :
+    F.mapᵣ A (f ≫ g) = F.mapᵣ A f ≫ F.mapᵣ A g := by
+  simp [Functor.mapᵣ]
+
+@[simp, reassoc]
+lemma Functor.mapᵣ_id (F : C ⥤ D ⥤ E) (A : C) {X : D} :
+    F.mapᵣ A (𝟙 X) = 𝟙 (F.obj₂ A X) := by
+  simp [Functor.mapᵣ]
```

Comment:
The fact that `simpNF` linter complains seems to confirm that it’s the "things in subterm" issue: for `Functor.mapᵣ` this does unify with `Functor.map_comp` and `Functor.map_id`, so `simp` finds them...